API.md

```diff
@@ -575,3 +577,146 @@ events to signal `'Unauthorized'` and `'Not Found'` (respectively) to the client
 '{"classification":{"nomination":{"author": "A friend"},"category":"Novel","canon_id":13}}'
 …
 ```
+
+
+## Raami exhibition
+
+### `GET /api/raami/artists`
+
+List of members who have opted to participate in the art exhibition
+
+#### Response
+```
+[
+  { person\_id: 'person_id',
+    continent: 'Europe',
+    }
+  …
+]
+```
+
+### `GET /api/raami/artists/:id`
+
+- Parameters: `id` (required)
+
+Full details for singular artist.
+
+#### Response
+```
+  { continent: 'Europe',
+    url: 'http://www.example.com',
+    filename: 'portfolio.pdf',
+    portfolio: [bytes],
+    category: 'Cover artist',
+    orientation: 'Fantasy',
+    description: 'Who is the artist etc.',
+    trasnport: 'Air mail' 
+    }
+  …
+```
+
+### `POST /api/raami/artist`
+
+- Parameters: `person_id`, `continent`, `url`, `filename`, `portfolio`, `category`, `orientation`, `description`, `transport`
+
+Insert new artist's details.
+
+#### Response
+```
+{
+  status: 'success',
+  inserted: [ 'id', 'continent', 'url','filename', 'portfolio', 'category', 'orientation', 'description', 'transport' ]
+}
```

Comment:
According to the implementation, `inserted` is in fact the id of the new artist.